pkg/resources/network_policy_acceptance_test.go

```diff
@@ -45,6 +45,16 @@ func TestAcc_NetworkPolicy(t *testing.T) {
 					resource.TestCheckResourceAttr("snowflake_network_policy.test", "blocked_ip_list.#", "1"),
 				),
 			},
+			// CHANGE PROPERTIES - empty ip lists
```

Comment:
I have written a similar comment to one of the recent @sfc-gh-jcieslak changes: https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/2743#discussion_r1582723559. Making the lists empty in this step will make the import step test incomplete/different. Ideally we should test all following situations:
- importing with "things" in every list and field
- importing a smaller subset
- creating object with lists filled
- updating object - setting lists to empty
- updating object - adding/removing to/from lists

So I am not sure if there is a nice ordering so that we can test it in just one test in different steps but pleasenot remove before impot :)

Reply:
Fixed, I added import check after setting non-empty lists.